deploy/helm/templates/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
is there a way to disable creating service? I would imagine not everyone wants to create a nodeport, so maybe should be configurable?